data/json/monsters.json

```diff
@@ -5965,6 +5965,34 @@
     "description":"Mom?",
     "flags":["SEES", "HEARS", "NO_BREATHE", "SMELLS", "GUILT"]
     },{
+    "type": "MONSTER",
+    "id": "mon_hallu_mannequin",
+    "name": "mannequin",
+    "species": "HALLUCINATION",
+    "default_faction": "factionless",
+    "symbol":"@",
+    "color":"ltgray",
+    "size":"MEDIUM",
+    "material":"plastic",
+    "diff":0,
+    "aggression":50,
+    "morale":100,
+    "speed":90,
+    "melee_skill":3,
+    "melee_dice":0,
+    "melee_dice_sides":0,
+    "dodge":1,
+    "armor_bash":2,
+    "armor_cut":4,
+    "armor_acid":6,
+    "vision_day":35,
+    "vision_night":5,
+    "hp":80,
+    "death_function":"DISAPPEAR",
+    "special_attacks":[["GRAB", 30]],
+    "description":"It moves in spurts when you blink, or at least you think it does.",
```

Comment:
This description implies something like a pounce attack, and bouncing around like that would make it even more confusing.